content/blog/tdd-2025/index.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+output: hugodown::hugo_document
+
+slug: tdd-2025
+title: Tidyverse developer day 2025
+date: 2025-06-24
+author: Max Kuhn
+description: >
+     Join us in Atlanta for tidyverse developer day on September 19, 2025!
+
+photo:
+  author: George Stagg
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [other] 
+tags: [tidyverse-dev-day]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The 2025 Tidy Dev Day will be on September 19th! We'll gather after posit::conf at the TODO from TODO to TODO. 
+
+Tidy Dev Day is an event where developers and open source contributors meet to collaboratively solve issues. You can hone your software engineering skills or work on your first contribution. We will have a slate of existing issues that you can work on with or without the assistance of our developers. 
+
+Here's how [Laura Acion](https://lacion.rbind.io/post/my-first-user/) described the 2019 event:
+
+> I wish there were more of these R-related dev days. Developing events that are friendly, diverse, and safe to advance not only the tidyverse. I see no downsides. Kuddos to the organizers.
+
+Plus...
+
+<div style="text-align: center">
+<img src="GONG.jpg" alt="plot of chunk gong" width="70%" />
+</div>
+
+<br>
+
+## Who should attend
+
+You should attend if: 
+
+- You have a laptop. 
+- You have an interest in contributing to the tidyverse, r-lib, and/or tidymodels repositories. 
+
+You **don't** have to have any previous development background or experience solving issues or making pull requests. We'll help you select an appropriate task from our pool of issues and assist you in getting started with GitHub, forks, pull requests, etc. You just need to bring enthusiasm (and a laptop helps too). The tidy teams will be on hand to help out, answer your questions, or get you started.
+
+## How to sign up
+
+There is a $TODO cost for the day; we provide the venue, snacks, lunch, and a barista. If cost is an issue, please email TODO and we'll work that out. We only charge a small fee to encourage people to show up for the day.
```

Comment:
Historically I feel like we donated some or all of the funds? It certainly doesn't cover the cost of the event. As you say, it's to discourage folks from flaking out. If donation is the current plan, we could mention the target org here.

Reply:
I saw that in a previous post but I had not heard anything about this year. I'm happy to put something in if we have confirmation.

Reply:
Yeah, we don't donate any more. It goes into Posit's general funds to slightly offset the cost of the event.